### ISSUE TITLE
Event languages limited be to a subset of organizer languages

### ISSUE THREAD

**Issue by MaicoTimmerman** [pretix/pretix#2105]
Currently, the organizer languages are only used on the organizer page. However, I'd like for events to not be created in a language that the organizer does not support. This also gives organizers the ability to disable languages for event creators in their organization.

I think this should only impact two places:
- Event creation wizard
- Event localization wizard

**Comment by raphaelm** [comment]
> Currently, the organizer languages are only used on the organizer page. However, I'd like for events to not be created in a language that the organizer does not support. This also gives organizers the ability to disable languages for event creators in their organization.

I'm not sure this is a good idea. First of all, it's highly backwards-incompatible. Second, it would require the event creation wizard to become a 4-step process instead of a 3-step process (or a validation error which isn't very user-friendly), since organizer and language are currently in the same step. Third, I have seen organizations only add an additional language for a specific event and forcing them to enable it organizer-wide just because of that… I see no good reason for that restriction? Am I missing something?

**Comment by MaicoTimmerman** [comment]
Our events are always available in the native language (dutch) and english. I'd prefer our users to not have to deal with selecting the correct languages, especially since there are two kinds of Dutch. Myself as a native dutch speaker am still unsure what is the difference between Dutch and Dutch (informal). To ensure uniformity we want all events to have the same languages. 

A suggestions that might help and I think is a nice (backward-compatible) UX improvement anyway:
- Set the event wizard locales initial value:
```py
        if len(self.fields['organizer'].choices) == 1:
            self.fields['organizer'].initial = self.fields['organizer'].queryset.first()
            # Add this line:
            self.fields["locales"].initial = self.fields['organizer'].queryset.first().locales
```

Another solution that would be even better for us is to add a new key `enabled` to the `[language]` table in `pretix.cfg` such that we can set `LANGUAGES` from the pretix.cfg:

```cfg
[languages]
enabled=nl,en
```

**Comment by raphaelm** [comment]
I think I'd be happy to merge both of these ideas!

**Comment by MaicoTimmerman** [comment]
This issue has been resolved with #2108 and #2109
